apps/notifications/views/crud.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from apps.notifications.models.payload import Payload
+from apps.notifications.serializers.payload import PayloadSerializer
+
+
+class NotificationPayloadCRUDView(APIView):
+    def get(self, request):
+        payloads = Payload.objects.all()
+        payload_serializer = PayloadSerializer(payloads, many=True)
+        return Response(payload_serializer.data)
+
+    def post(self, request):
+        payload_serializer = PayloadSerializer(data=request.data)
+        if payload_serializer.is_valid():
+            payload_serializer.save()
```

Comment:
Another way of doing this is 
`payload_serializer.is_valid(raise_exception=True)`
This way we don't need to add an if condition

Reply:
Oh cool!